lib/helpers.bash

```diff
@@ -849,3 +849,22 @@ then
     fi
   }
 fi
+
+function _bash_it_find_in_ancestor() (
```

Comment:
I'm still getting used to the [coding standards everyone agreed to](https://bash-it.readthedocs.io/en/latest/contributing/#code-style), so I'm sorry I didn't bring this up sooner. Could you switch this to hyphens?

`_bash-it-find-in-ancestor`

Reply:
Using hyphens is super weird. Hyphens are considered by _Bash_ and most everywhere else to be word breaks. 

If that's really how y'all want it, do you want me to alsö start changing other function names as I continue through the code? What about the leading underscore for not-user-facing functions?

Reply:
@cornfeedhobo, is there a better way to address this weird? I don't particularly care for one PR, but this is a weird across the whole `lib/helpers`. should I open a separate issue? Was there a whole discussion on it I can read through before wasting my time?

Reply:
I think you should open an issue about it @gaelicWizard